### PR TITLE
Various docs improvements

### LINKED ISSUE #142: Add link to overview page
On the Using Radicle / Overview page `add other people as remotes to your project` should link to the corresponding page where we describe how to add a remote to a project.

### PR DESCRIPTION
**Pull request by brandonhaslegs**
Fixed the search bar and styled the search results to look more Radicle. Removed the "Back to radicle.xyz" button because it just seems weird. This is another website, it doesn't need to link back to radicle.xyz so prominently.

### CODE REVIEW

File: docs/using-radicle/overview.md [hunk 1]
```diff
@@ -60,4 +60,5 @@ For more help on using Radicle, be sure to join our [community channels][cc].
 [dr]: radicle.md/#how-is-radicle-different-from-github
 [gi]: understanding-radicle/how-it-works.md
 [rl]: https://github.com/radicle-dev/radicle-link
+[arm]: https://docs.radicle.xyz/docs/using-radicle/tracking-and-viewing#adding-remotes
```
Comment (rudolfs) [inline docs/using-radicle/overview.md:63]:
```suggestion
[ar]: using-radicle/tracking-and-viewing#adding-remotes
```

File: docs/using-radicle/overview.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ its maintainers and contributors, instead of being confined to the control of a
 small group of people with read/write access.
 
 In practice, this means that to fetch and receive changes from contributors, you
-have to **add other people as remotes to your project**. Adding someone as a
+have to [add other people as remotes to your project][arm]. Adding someone as a
```
Comment (rudolfs) [inline docs/using-radicle/overview.md:20]:
```suggestion
have to [add other people as remotes to your project][ar]. Adding someone as a
```

**Comment by brandonhaslegs** [comment]
The only thing I can't figure out is how to style the mobile search results. They disappear when I inspect them so I can't figure out the styles that are making them look that way. They just need 1rem padding and 1rem border-radius.

image

**Comment by brandonhaslegs** [comment]
@rudolfs If you find a better way of styling without using !important everywhere, I'd love to know.

File: docs/using-radicle/overview.md [hunk 3]
```diff
@@ -60,4 +60,5 @@ For more help on using Radicle, be sure to join our [community channels][cc].
 [dr]: radicle.md/#how-is-radicle-different-from-github
 [gi]: understanding-radicle/how-it-works.md
 [rl]: https://github.com/radicle-dev/radicle-link
+[ar]: using-radicle/tracking-and-viewing#adding-remotes
```
Comment (rudolfs) [inline docs/using-radicle/overview.md:63]:
```suggestion
[ar]: using-radicle/tracking-and-viewing.md#adding-remotes
```

**Review by rudolfs — APPROVED** [review]

**Comment by rudolfs** [comment]
> If you find a better way of styling without using !important everywhere, I'd love to know.

Don't think it's possible in this old version of Docusaurus that we're using. Newer versions seem to have proper styling overrides, but we'd have to upgrade.

https://docusaurus.io/docs/search#styling-your-algolia-search

**Pull request merged**